src/legacy/core_plugins/metrics/public/components/annotations_editor.js

```diff
@@ -29,8 +29,7 @@ import uuid from 'uuid';
 import { IconSelect } from './icon_select';
 import { YesNo } from './yes_no';
 import { Storage } from 'ui/storage';
-import { data } from 'plugins/data/setup';
-const { QueryBarInput } = data.query.ui;
+import { QueryBarInput } from 'plugins/data';
```

Comment:
Since we are already touching all of the imports, we should consider switching everything over to relative paths for consistency (including our dev docs notes).

Reply:
Same as above, lets wait for @epixa solution https://github.com/elastic/kibana/issues/40446 to be ready and make a separate PR for this.

Reply:
Yeah, anyway, changing the paths to relative-from-src is possible ATM only for types, but not for classes \ functions, as webpack is not configured correctly yet.

Pushing all other updates and we'll make a separate PR for this.